api/src/main/java/org/sakaiproject/content/api/ContentChangeHandlerImpl.java

```diff
@@ -0,0 +1,7 @@
+package org.sakaiproject.content.api;
+
+public class ContentChangeHandlerImpl implements ContentChangeHandler {
```

Comment:
If you're doing this for interface evolution Java 8 default methods go some way to handling this and might be better?